### PR TITLE
PRTL-902 Set height of cancer distribution histogram to 300. Remove bar width scaling

### PR DESCRIPTION
**Pull request by cheapsteak**
Have confirmed that we indeed want to remove bar width scaling based on number of bars 
https://oicr-gdc.slack.com/archives/C0H0726P7/p1490623679788124

### CODE REVIEW

**Comment by alex-wilmer** [comment]
maybe this is a good opportunity to remove that code from the individual charts? would it be possible to handle this stuff one time in `charts/BarChart.js` ?

**Comment by cheapsteak** [comment]
I'd recommend keeping it here for a release or until we're certain that this spec wont get reverted

File: modules/node_modules/@ncigdc/containers/FrequentMutationsChart.js [hunk 1]
```diff
@@ -41,9 +41,10 @@ const FrequentMutationsChartComponent = compose(
 }) => {
   const chartMargin = { top: 30, right: 50, bottom: 105, left: 40 };
 
+  const maxBars = 20;
   const bandWidth = ssms && (
```
Comment (alex-wilmer) [inline modules/node_modules/@ncigdc/containers/FrequentMutationsChart.js]: can remove these null checks

Comment (cheapsteak) [inline modules/node_modules/@ncigdc/containers/FrequentMutationsChart.js]: removed

**Pull request merged**